test/integration/good/code-gen/cpp.expected

```diff
@@ -457,6 +457,816 @@ class eight_schools_ncp_model : public model_base_crtp<eight_schools_ncp_model>
 
 typedef eight_schools_ncp_model_namespace::eight_schools_ncp_model stan_model;
 
+// Boilerplate
```

Comment:
Is it ok to add another glm model to the regular test suite?

Or should we just move all models that will be added to test --use-opencl to a separate folder and have:
-  cpp-no-cl.expected (to test we dont break anything if --use-opencl is not on) 
- cl.expected (--use-opencl on)